caches/infinispan/src/main/java/org/commonjava/maven/galley/cache/infinispan/NFSOwnerCacheProducer.java

```diff
@@ -55,7 +55,7 @@ public Configuration getCacheCfg()
     protected EmbeddedCacheManager getCacheMgr()
     {
         final EmbeddedCacheManager cacheManager = new DefaultCacheManager(
-                new GlobalConfigurationBuilder().globalJmxStatistics().jmxDomain( "org.commonjava" ).build() );
+                new GlobalConfigurationBuilder().globalJmxStatistics().jmxDomain( "org.commonjava.maven.galley" ).build() );
```

Comment:
If we do not specify a jmx domain here, it will always report an exception said something like  "ISPN000034: There's already a JMX MBean instance type=CacheManager,name="CDIExtensionDefaultCacheManager" already registered under 'org.infinispan' JMX domain.".